8.0.0-rc.0 requires zone.js version ~0.9.0

<!--🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅

Oh hi there! 😄

To expedite issue processing please search open and closed issues before submitting a new one.
Existing issues often contain information about workarounds, resolution, or progress updates.

🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅-->


# 🐞 Bug report

### Command (mark with an `x`)
<!-- Can you pin-point the command or commands that are effected by this bug? -->
<!-- ✍️edit: -->
```
- [ ] new
- [x] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?

<!-- Did this behavior use to work in the previous version? -->
Yes, the previous version in which this bug was not present was: last one.


### Description

<!-- ✍️--> A clear and concise description of the problem...
I have an existing project whose package.json had "zone.js": "^0.8.29" dependency. Looks like the switch to the evergreen version now requires ~.0.9.0


## 🔬 Minimal Reproduction
ng build --production with "zone.js": "^0.8.29" gives the following error.

ERROR in ./src/polyfills.ts
Module not found: Error: Can't resolve 'zone.js/dist/zone' in '/Users/jimoneill/WebstormProjects/commerce/src'


<!--
Simple steps to reproduce this bug.

Please include: commands run (including args), packages added, related code changes.

If reproduction steps are not enough for reproduction of your issue, please create a minimal GitHub repository with the reproduction of the issue. 
A good way to make a minimal reproduction is to create a new app via `ng new repro-app` and add the minimum possible code to show the problem. 
Share the link to the repo below along with step-by-step instructions to reproduce the problem, as well as expected and actual behavior.

Issues that don't have enough info and can't be reproduced will be closed.

You can read more about issue submission guidelines here: https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue
-->

## 🔥 Exception or Error
<pre><code>
<!-- If the issue is accompanied by an exception or an error, please share it below: -->
<!-- ✍️-->

</code></pre>


## 🌍 Your Environment
<pre><code>
<!-- run `ng version` and paste output below -->
<!-- ✍️-->

</code></pre>

**Anything else relevant?**
<!-- ✍️Is this a browser specific issue? If so, please specify the browser and version. -->

<!-- ✍️Do any of these matter: operating system, IDE, package manager, HTTP server, ...? If so, please mention it below. -->

--- a/packages/schematics/angular/migrations/update-8/update-builders.ts
+++ b/packages/schematics/angular/migrations/update-8/update-builders.ts
@@ -42,6 +42,20 @@ export function updateBuilders() {
       );
     }
 
+    current = getPackageJsonDependency(host, 'zone.js');
+    if (current && current.version !== latestVersions.ZoneJs) {
+      updates = true;
+      addPackageJsonDependency(
+        host,
+        {
+          type: current.type,
+          name: 'zone.js',
+          version: latestVersions.ZoneJs,
+          overwrite: true,
+        },
+      );
+    }
+
     if (updates) {
       context.addTask(new NodePackageInstallTask());
     }

